### ISSUE TITLE
xi:fallback does not work

### ISSUE THREAD

**Issue by weirdan** [vimeo/psalm#7243]
`<xi: fallback>` is supposed to kick in when the included file is not available, but Psalm instead throws the following exception:

```
Uncaught RuntimeException: PHP Error: DOMDocument::xinclude(): I/O warning : failed to load external entity "/home/weirdan/src/psalm/psalm/zz.xml" in /home/weirdan/src/psalm/psalm/src/Psalm/Config.php:694 in /home/weirdan/src/psalm/psalm/src/Psalm/Internal/ErrorHandler.php:66
Stack trace:
#0 [internal function]: Psalm\Internal\ErrorHandler::Psalm\Internal\{closure}()
#1 /home/weirdan/src/psalm/psalm/src/Psalm/Config.php(694): DOMDocument->xinclude()
#2 /home/weirdan/src/psalm/psalm/src/Psalm/Config.php(711): Psalm\Config::loadDomDocument()
#3 /home/weirdan/src/psalm/psalm/src/Psalm/Config.php(679): Psalm\Config::validateXmlConfig()
#4 /home/weirdan/src/psalm/psalm/src/Psalm/Config.php(644): Psalm\Config::loadFromXML()
#5 /home/weirdan/src/psalm/psalm/src/Psalm/Config.php(596): Psalm\Config::loadFromXMLFile()
#6 /home/weirdan/src/psalm/psalm/src/Psalm/Internal/CliUtils.php(494): Psalm\Config::getConfigForPath()
#7 /home/weirdan/src/psalm/psalm/src/Psalm/Internal/Cli/Psalm.php(563): Psalm\Internal\CliUtils::initializeConfig()
#8 /home/weirdan/src/psalm/psalm/src/Psalm/Internal/Cli/Psalm.php(1009): Psalm\Internal\Cli\Psalm::loadConfig()
#9 /home/weirdan/src/psalm/psalm/src/Psalm/Internal/Cli/Psalm.php(231): Psalm\Internal\Cli\Psalm::initConfig()
#10 /home/weirdan/src/psalm/psalm/psalm(7): Psalm\Internal\Cli\Psalm::run()
#11 {main}
(Psalm dev-master@1c078136273a669d52d234251ddbae4cd0507d38 crashed due to an uncaught Throwable)
```

when `<xi:include>` is used in `issueHandlers` element:

```xml
        <xi:include href="zz.xml">
            <xi:fallback>
                <MixedAssignment>
                    <errorLevel type="suppress">
                        <directory name="vendor/nikic/php-parser" />
                    </errorLevel>
                </MixedAssignment>
            </xi:fallback>
        </xi:include>
```

**Comment by orklah** [comment]
Yeah, please do :)

**Comment by weirdan** [comment]
Fixed in #7324
